test/Dialect/Torch/func-builtin-tensorize.mlir

```diff
@@ -0,0 +1,94 @@
+// RUN: npcomp-opt %s -torch-func-builtin-tensorize -split-input-file -verify-diagnostics -allow-unregistered-dialect | FileCheck %s
```

Comment:
Am I thinking about this wrong to expect a test in here with the pass operating on a !torch.tensor (even if just to verify that the wrong thing isn't done)?

Reply:
We don't do anything specific to !torch.tensor, so it just passes through trivially (arguably we should make it an error, in which case we should add a test). In effect it would just be a redundant test of the func conversion patterns, or equivalent to a test with an i64 or other random type argument (we actually already have one like that).